framework/src/modules/chain/blocks/fork_choice_rule.js

```diff
@@ -142,12 +142,17 @@ const isTieBreak = ({ slots, lastAppliedBlock, receivedBlock }) =>
  * @return {boolean}
  */
 // eslint-disable-next-line class-methods-use-this
-const isDifferentChain = (lastBlock, currentBlock) =>
-	lastBlock.prevotedConfirmedUptoHeight <
-		currentBlock.prevotedConfirmedUptoHeight ||
-	(lastBlock.height < currentBlock.height &&
-		lastBlock.prevotedConfirmedUptoHeight ===
-			currentBlock.prevotedConfirmedUptoHeight);
+const isDifferentChain = (lastBlock, currentBlock) => {
+	if (lastBlock.height === 1) return true;
```

Comment:
As coding standard, it should be
```
if (lastBlock.height === 1) {
  return true;
}
```
Also, why if the last block height is 1, then it's different chain?